### PR TITLE
YES-app

### PR DESCRIPTION
**Pull request by BrandonMFong**
Just making a pull request to let you know that I am merging.  Will be deleting these branches

Your app works.

### CODE REVIEW

File: YES-app/YES/templates/index-layout.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            <a class="nav-link text-white" href="{% url 'about' %}">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-white" href="{% url 'login' %}">Log in</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-white" href="{% url 'accounts:signup' %}">Sign Up</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-white" href="https://dualpowergeneration.sdsu.edu/datacenter/">Data Center</a>
```
Comment (BrandonMFong) [inline YES-app/YES/templates/index-layout.html]: @alrabdullah this links to the already existing site.  I will take care of the data center webpage

Comment (BrandonMFong) [inline YES-app/YES/templates/index-layout.html]: @alrabdullah this links to the site on the sdsu server. I will take care of developing the datacenter web page

**Comment by BrandonMFong** [comment]
Just assigning all of you here to grab your attention so that you can let abdullah know of this. 

@team I am merging his site and will continue the work on the data center on my own.

**Comment by CharlesSotto** [comment]
Forsure I’ll do my best

**Pull request merged**